src/main/resources/db/migration/0200/V225__PlantingSiteNotifications.sql

```diff
@@ -0,0 +1,55 @@
+CREATE TABLE tracking.planting_site_notifications (
+    id BIGINT PRIMARY KEY GENERATED BY DEFAULT AS IDENTITY,
+    planting_site_id BIGINT NOT NULL REFERENCES tracking.planting_sites ON DELETE CASCADE,
+    notification_type_id INTEGER NOT NULL REFERENCES notification_types,
```

Comment:
should we validate that notification type is one of  schedule observation, observation not scheduled (support)?